lib/formatters/html.js

```diff
@@ -82,30 +83,32 @@ function renderMessages(messages, parentIndex) {
             severityNumber: message.severity,
             severityName: message.severity === 1 ? "Warning" : "Error",
             message: message.message,
-            ruleId: message.ruleId
+            ruleId: message.ruleId,
+            url: rules.get(message.ruleId).meta.docs.url
```

Comment:
Can you check this to ensure that `meta.docs` and `meta.docs.url` are not null?

Reply:
Good catch. That would have broken on rules where `meta.docs` is not defined. (I'm assuming that's what you meant.)